lib/next_ls/commands/from_pipe.ex

```diff
@@ -0,0 +1,116 @@
+defmodule NextLS.Commands.FromPipe do
+  @moduledoc false
+  alias GenLSP.Enumerations.ErrorCodes
+  alias GenLSP.Structures.Position
+  alias GenLSP.Structures.Range
+  alias GenLSP.Structures.TextEdit
+  alias GenLSP.Structures.WorkspaceEdit
+
+  defp opts do
+    Schematic.map(%{
```

Comment:
Please let me know if you have any feedback for the Schematic library! It can get the job done but it was first built with codegen in mind, so the ergonomics are a little off I think. Curious to hear your thought!